emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+  </div>
+  <table style="vertical-align: top;min-width: 320px;margin: 0 auto;width:100%" cellpadding="0" cellspacing="0">
+    <tbody>
+      <tr style="vertical-align: top">
+        <td style="word-break: break-word;border-collapse: collapse ;vertical-align: top">
+
+          <!-- header -->
+          <table id="header-section" width="100%" border="0" style="max-width: 850px; display: block;margin:0 auto;">
+              <tbody style="display: block;">
+                <tr style="display: block;">
+                  <td style="display: block;"> 
+                    <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/relay-wordmark.png" alt="" title="" style="height: auto;float: none;width: 100%;max-width: 125px; margin: 10px auto; display: block;" /> 
+                  </td>
+                </tr>
+              </tbody>
+          </table>
+
+          <!-- hero section -->
+          <table id="hero-section" style="max-width: 850px; display: block;margin:0 auto;background-color: #321c64" cellpadding="0" cellspacing="0" width="100%" border="0">
+            <tbody>
+              <tr>
+                <td align="center"> 
+                  <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/hero-image-en.png" alt="" title="" /> 
+                </td>
+
+                <td style="overflow-wrap:break-word;word-break:break-word;padding:10px;" align="left"> 
+                  <h1 class="hero-title" style="margin: 0px; color: #ffffff; line-height: 100%; text-align: left; word-wrap: break-word; font-weight: 500px;">{% ftlmsg 'first-time-user-email-welcome' %}</h1>  
+                  <p class="hero-text" style="line-height: 140%; color: #ffffff;font-family: metropolis medium">{% ftlmsg 'first-time-user-email-welcome-subhead' %}</p>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+
+          <!-- what you get -->
+          <table id="what-is-relay-section" style="max-width: 850px; display: block;margin:0 auto;" cellpadding="0" cellspacing="0" width="100%" border="0">
+            <tbody>
+              <tr>
+                <td style="overflow-wrap:break-word;word-break:break-word;" align="center">
+
+                  <p style="color: #321C64; font-size: 18px; font-family: metropolis medium;">{% ftlmsg 'first-time-user-email-hero-primary-text' %}</p> 
+                  <p style="color: #321C64; font-size: 18px; font-family: metropolis medium;margin-top: 20px;">{% ftlmsg 'first-time-user-email-hero-secondary-text' %}</p>
+
+                  <a href="{{ SITE_ORIGIN }}/accounts/profile/?utm_campaign=first_email&utm_source=email&utm_medium=email" target="_blank" class="button" style="margin-top: 30px;" rel="noreferrer">{% ftlmsg 'first-time-user-email-hero-cta' %}</a>
+
+                </td>
+              </tr>
+            </tbody>
+          </table> 
+
+          <!-- separator -->
```

Comment:
I'm assuming `<hr>`s are unreliable or not stylable enough in email clients? :exploding_head:

Reply:
Correct